Gems/PhysX/Code/Editor/ComboBoxEditButtonPair.h

```diff
@@ -21,11 +21,23 @@ namespace PhysX
         class ComboBoxEditButtonPair
```

Comment:
Maybe outside the scope here, but this class is actually duplicated in the `NvCloth` gem. We should either move it to `AzQtComponents` so it can have a single class, or could remove it all-together and just update the existing combobox handler to add support for an edit icon/action.

Reply:
Might be nice to do as a follow-up PR I'd say